micro_dl/input/training_table.py

```diff
@@ -86,7 +86,7 @@ def train_test_split(self):
             "into train/val".format(self.split_by_column)
         # DOES NOT HANDLE NON-INTEGER VALUES. map to int if string
         # the sample_idxs are required for evaluating performance on test set
-
+        unique_values = unique_values.astype(np.uint32)
```

Comment:
wondering what is the rationale for this? this would make the assert below unnecessary. If unique_values is not an integer (say label_names), it will not raise an exception!

Reply:
does pandas unique() not return an int array? I never checked that...

Reply:
At some point I was getting indices in float type when I was debugging so I added this as a temporary fix... If the input are all integers pandas unique() should return integers. I'll remove it!